docs/reST/ref/key.rst

```diff
@@ -427,4 +427,28 @@ for ``KMOD_NONE``, which should be compared using equals ``==``). For example:
 
    .. ## pygame.key.set_text_input_rect ##
 
+.. function:: get_just_pressed
+
+   | :sl:`returns a tuple containing the most recent key presses`
+   | :sg:`get_just_pressed() -> bools`
+
+   Returns a sequence of boolean values representing the most recent key presses
+   which are fetched through the use of pygame.event.get()
```

Comment:
This explanation could use a bit more clarity about how/why it's interacting with pygame.event.get. Also would be great to have a small code sample (just a couple lines) that shows how it works.